src/main/java/io/aiven/connect/elasticsearch/Mapping.java

```diff
@@ -67,7 +67,7 @@ public static JsonObject getMapping(final ElasticsearchClient client, final Stri
      *
      * @param schema The schema used to infer mapping.
      */
-    public static JsonNode inferMapping(final ElasticsearchClient client, final Schema schema) {
+    public static JsonNode inferMapping(final ElasticsearchClient.Version version, final Schema schema) {
```

Comment:
not blocking, I think this should be unit tested and we should compare the JsonNode coming out from before and after the update (to both help us keeping the backcompatibility and also to figure out faster which data do we send to elasticsearch with a certain input)